### ISSUE TITLE
KeyError: 'collection' hot-fix

### ISSUE THREAD

**Issue by SantaSpeen** [felix-hilden/tekore#270]
### Issue
tekore.Spotify("token").playback(tracks_only=True)
raisying `KeyError: 'collection'`

### Hot fix
```python
from tekore._model import context, serialise

class ContextType(serialise.StrEnum):
    """Type of player context."""

    album = 'album'
    artist = 'artist'
    playlist = 'playlist'
    show = 'show'
    collection = 'collection'

context.ContextType = ContextType
```

* Need to add     `collection = 'collection'` to `ContextType(StrEnum)`

**Comment by felix-hilden** [comment]
Oh, thanks for submitting! I'll have a look.

**Comment by HarrySky** [comment]
Some context (pun intended) - it occurs when you are requesting currently listening song and listening from "Liked Songs" list. They changed context for "Liked Songs", it was `None` before and now it seems that context exists and has `collection` type

**Comment by felix-hilden** [comment]
That'll probably help a bunch with chasing it down, thanks! Working on it now.

**Comment by felix-hilden** [comment]
Verified and fixed, I'll release after including #267.
